contracts/src/OutputOracle.sol

```diff
@@ -169,7 +169,7 @@ contract OutputOracle is Initializable, ISemver {
         // Make sure an output for this block number has actually been proposed.
         require(
             _l2BlockNumber <= latestBlockNumber(),
-            "L2OutputOracle: cannot get output for a block that has not been proposed"
```

Comment:
keeping the original name in errors was mostly intentional, as it maintains the same error with the original `L2OutputOracle` implementation